packages/schematics/angular/utility/json-utils.ts

```diff
@@ -31,6 +37,44 @@ export function appendPropertyInAstObject(
   );
 }
 
+export function removePropertyFromAstObject(
```

Comment:
This is unused, and if we were to use it we'd probably want some tests on it. Could you remove it quickly?

Reply:
Removed.